official-allow_functionality.json

```diff
@@ -229,8 +229,43 @@
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.s3-us-west-2.amazonaws.com"}},
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.cloudflare.com"}},
 		{"o":{"h":"*.hansje.net"},"d":{"h":"*.cargocollective.com"}},
-		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}}
+		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}},
+		{"o":{"h":"*.eurobuch.com"},"d":{"h":"*.lehmanns.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.acfrg.com"}},
+		{"o":{"h":"*.microsoft.com"},"d":{"h":"*.s-msft.com"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.trackersimulator.org"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.eviltracker.net"}},
```

Comment:
I meant the rule should be

     +		{"o":{"h":"panopticlick.eff.org"},"d":{"h":"*.trackersimulator.org"}}, 

not the wildcard `*.eff.org`. But yes we might want to remove these rules as this demonstrates how RP can help preventing fingerprinting.